doc/itof/nxp_build_instructions.md

```diff
@@ -68,6 +69,17 @@ sudo cmake --build . --target install
 popd
 ```
 
+* Depth Compute:
+The Depth Compute library is required to build the SDK and run any application based on the SDK.
+Based on the following scenarios you will be able to do the following:
+
+1. You have the **development package** (tofi_compute.dll, tof_compute.lib, tofi_config.dll, tofi_config.lib) for the Depth Compute library. With it you will be able to build the SDK. You just need to put the files mentioned above in a directory called 'libs' which should be placed at the same level as the repository directory.
```

Comment:
You can mention that the users need to use the arm development package for nxp. And the depth libraries are .so files, not dll's.

Reply:
Solved